src/dsp_tools/commands/xmlupload/resource_create_client.py

```diff
@@ -41,7 +42,10 @@ def create_resource(
         )
         resource_dict = self._make_resource_with_values(resource, bitstream_information)
         resource_json_ld = json.dumps(resource_dict, ensure_ascii=False)
-        res = self.con.post(route="/v2/resources", jsondata=resource_json_ld)
+        if self.media_previously_ingested:
+            res = self.con.post(route="/v2/resources", jsondata=resource_json_ld, headers={"X-Asset-Ingested": "true"})
+        else:
+            res = self.con.post(route="/v2/resources", jsondata=resource_json_ld)
```

Comment:
```suggestion
        headers = {"X-Asset-Ingested": "true"} if self.media_previously_ingested else None # or {} respectively, see my other comments
        res = self.con.post(route="/v2/resources", jsondata=resource_json_ld, headers=headers)
```
I think this would look a bit nicer